source/changelog.txt

```diff
@@ -12,6 +12,28 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v1.5.4
+------
+
+*Released July 31, 2022*
+
+Fixes a potential data corruption bug in
+``KeyVault.rewrapManyDataKey()`` when rotating encrypted data encryption
+keys backed by Azure or GCP key services.
+
+In previous versions of ``mongosh``, this bug occurrs when an
+Azure-backed or GCP-backed data encryption key being rewrapped required
```

Comment:
I know this is the suggested text, but we should at least fix the tenses:

Either:
in previous versions of ``mongosh``, this bug occured when an Azure-backed or GCP-backed data encryption key that was being rewrapped required fetching an access token for decryption of the data encryption key.

Or:
n previous versions of ``mongosh``, this bug occurs when an Azure-backed or GCP-backed data encryption key that is being rewrapped requires fetching an access token for decryption of the data encryption key.

Reply:
Fixed